modules/manually-maintained-credentials-upgrade.adoc

```diff
@@ -8,20 +8,58 @@
 [id="manually-maintained-credentials-upgrade_{context}"]
 = Upgrading clusters with manually maintained credentials
 
-If credentials are added in a future release, the Cloud Credential Operator (CCO) `upgradable` status for a cluster with manually maintained credentials changes to `false`. For minor release, for example, from 4.7 to 4.8, this status prevents you from upgrading until you have addressed any updated permissions. For z-stream releases, for example, from 4.7.12 to 4.7.13, the upgrade is not blocked, but the credentials must still be updated for the new release.
+The Cloud Credential Operator (CCO) `upgradable` status for a cluster with manually maintained credentials is `false` by default.
 
-Use the *Administrator* perspective of the web console to determine if the CCO is upgradeable.
+* For minor releases, for example, from 4.7 to 4.8, this status prevents you from upgrading until you have addressed any updated permissions and annotated the `CloudCredential` resource to indicate that the permissions are updated as needed for the next version. This annotation changes the `upgradable` status to `true`.
 
-. Navigate to *Administration* -> *Cluster Settings*.
+* For z-stream releases, for example, from 4.7.12 to 4.7.13, no permissions are added or changed, so the upgrade is not blocked.
```

Comment:
the same issue as the above, is it better to change example `from 4.7.12 to 4.7.13` to `from 4.8.10 to 4.8.11`?